src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class LinuxAppServiceLogger
+    {
+        private const string ConsolePrefix = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
+        private const string LoggingPath = "/var/log/functionsLogs/durableevents.log";
+        private readonly bool writeToConsole;
+
+        public LinuxAppServiceLogger(bool writeToConsole)
+        {
+            // If writeToConsole is False, we write to a file
+            this.writeToConsole = writeToConsole;
+        }
+
+        public string GenerateJson<TState>(TState maybeEventData)
```

Comment:
Now that we've removed the unnecessary abstractions, let's remove the `TState` type parameter and just use `EventWrittenEventArgs` directly.

Reply:
Yeah I should have caught on to this. thanks